xml/security_ldap.xml

```diff
@@ -889,7 +663,258 @@ telephoneNumber: +49 1234 567-10</screen>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="ldap_y2_adset.png" width="75%" format="PNG"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="ldap_y2_adset.png" width="75%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  -->
+
+  <para>
+   The initial input form of user administration offers <guimenu>LDAP
+   Options</guimenu>. This allows you to apply LDAP search filters to the
+   set of available users. Alternatively open the module for configuring
+   LDAP users and groups by selecting <guimenu>LDAP User and Group
+   Configuration</guimenu>.
+  </para>
+ </sect1>
+
+<sect1 xml:id="sec.security.ldap.yast.client">
+  <title>Configuring an LDAP Client with &yast;</title>
+<!--taroth 2019-02-25: let's see where to move/integrate the content from this
+    section in the end-->
+  <para>
+   &yast; includes the module <guimenu>LDAP and Kerberos Client</guimenu>
+   that helps define authentication scenarios involving either LDAP or Kerberos.
+  </para>
+  <para>
+   It can also be used to join Kerberos and LDAP separately. However, in
+   many such cases, using this module may not be the first choice, such as
+   for joining &ad; (which uses a combination of LDAP and Kerberos). For
+   more information, see <xref linkend="sec.security.auth.yast.client"/>.
+  </para>
+  <para>
+   Start the module by selecting
+   <menuchoice> <guimenu>Network Services</guimenu> <guimenu>LDAP and
+   Kerberos Client</guimenu> </menuchoice>.
+  </para>
+  <figure xml:id="fig.yast2.ldapkerberos.ldap">
+   <title><guimenu>LDAP and Kerberos Client</guimenu> Window</title>
+   <mediaobject>
+    <imageobject role="html">
+     <imagedata fileref="yast2_auth_client_config.png" width="65%" format="PNG"/>
+    </imageobject>
+    <imageobject role="fo">
+     <imagedata fileref="yast2_auth_client_config.png" width="65%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+  <para>
+   To configure an LDAP client, follow the procedure below:
+  </para>
+  <procedure xml:id="pro.security.auth.ldap">
+   <step>
+    <para>
+     In the window <guimenu>LDAP and Kerberos Client</guimenu>, click
+     <guimenu>Change Settings</guimenu>.
+    </para>
+    <para>
+     Make sure that the tab <guimenu>Use a Directory as Identity Provider
+     (LDAP)</guimenu> is chosen.
+    </para>
+    <informalfigure>
+     <mediaobject>
+      <imageobject role="html">
+       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
+      </imageobject>
+      <imageobject role="fo">
+       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
+      </imageobject>
+     </mediaobject>
+    </informalfigure>
+   </step>
+   <step>
+    <para>
+     Specify one or more LDAP server URLs, host names, or IP addresses under
+     <guimenu>Enter LDAP server locations</guimenu>. When specifying multiple
+     addresses, separate them with spaces.
```

Comment:
Recommend that the URLS be LDAPS:// only (startTLS and plain both have security issues)

Reply:
Done